services/processing/YrProcessor/serverless.yml

```
@@ -0,0 +1,122 @@
+    #infrastructure: s3
+    # key: value
+  stackTags: # Tags for cloud formation
+    # key: value
+  environment: # Service wide environment variables
+    #ENVIRONMENT_VARIABLE_NAME: 123456789
+
+package:
+  individually: true
+  exclude:
+    - "./**" # Exclude everything and include what you need in the functions below
+
+functions:
+  yrProcessor:
+    module: src/${self:custom.editable.folder} # path to function
+    handler: ${self:custom.editable.handlerFile}.handler # fileName.functionNameInFile
+    name: ${self:custom.stage}-${self:custom.editable.handlerFile} # Always begin name with stage
+    description: ${self:custom.editable.description}
+    role: !GetAtt S3Access.Arn
+    package:
+      include:
+        - 'src/${self:custom.editable.folder}/*.py' # Files to include in deployment
+    environment: # Environment variables, often stored in SSM
+      #ENVIRONMENT_VARIABLE_NAME: ${ssm:ssm_key}
+      STAGE: ${self:custom.stage}
+      DATABASE_ENDPOINT_ADDRESS: !ImportValue ${self:custom.stage}-DBEndpointAddress
+      DATABASE_ENDPOINT_PORT: !ImportValue ${self:custom.stage}-DBEndpointPort
+    tags: # Tag for this function. Every function are tagged with stage by default
+    events:
+      - s3: # Lambda fires when object is created in s3
+          bucket: !ImportValue ${self:custom.stage}-datalakeName
+          event: s3:ObjectCreated:*
+          rules: # OBject must be in within this path GoogleCalendarEvents/**.json
+            - prefix: ${self:custom.accessPath}
+            - suffix: .json
+          existing: true
+    vpc: # This is needed to access postgres
+      securityGroupIds:
+        - !ImportValue ${self:custom.stage}-PostgresAccessSecurityGroupID #This group have access to postgres
+      subnetIds:
+        - !ImportValue ${self:custom.stage}-SubnetAID
+        - !ImportValue ${self:custom.stage}-SubnetBID
+        - !ImportValue ${self:custom.stage}-SubnetCID
+
+
+resources: # The resources your functions use
+  Resources:
+    S3Access:
+      Type: AWS::IAM::Role
+      Properties:
+        RoleName: ${self:custom.editable.handlerFile}
+        AssumeRolePolicyDocument:
+          Version: "2012-10-17"
+          Statement:
+            - Effect: Allow
+              Principal:
+                Service:
+                  - lambda.amazonaws.com
+              Action: sts:AssumeRole
+        ManagedPolicyArns:
+          - "arn:aws:iam::aws:policy/service-role/AWSLambdaVPCAccessExecutionRole"
+          - "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"
+        Policies: # Give lambda access to s3 data
+          - PolicyName: ${self:custom.stage}-DatalakeRead-${self:custom.editable.handlerFile}
+            PolicyDocument:
+              Version: "2012-10-17"
+              Statement:
+                - Effect: Allow
+                  Action:
+                    - s3:GetObject
+                  Resource: # Only give access to the folder GoogleCalendarEvents/
```

Dette er en feil fra templaten
```suggestion
                  Resource:
```